### ISSUE TITLE
Packaging of mp3 encoded files using Bento4

### ISSUE THREAD

**Issue by prachi-rai** [axiomatic-systems/Bento4#526]
Hi, 
   Is it possible to do packaging of mp3 encoded files using Bento4.

**Comment by barbibulle** [comment]
Packaging mp3 elementary streams in MP4 containers is possible, but very rarely used. That's why it isn't currently implemented. Do you have a use case for it?

**Comment by haridarshan** [comment]
@barbibulle 
Yes, We've to adaptive stream mp3 audio files available in 64, 128, 192, 256 and 320 kbps by packaging them in mp4 containers for both dash and hls and possibly with drm protection as well

**Comment by hungvo2010** [comment]
Did you find a way to solve your issue?
